l10n/en/firefox/installer-help.ftl

```diff
@@ -15,15 +15,20 @@ installer-help-action-choose = Choose a download below.
 
 installer-help-firefox-release-title = { -brand-name-firefox-browser }
 
+# Variables:
+#   $trackers (number) - number of trackers blocked by Firefox (currently in the thousands)
+installer-help-firefox-release-desc-v2 = Get the latest. Automatic privacy is here. Download { -brand-name-firefox } to block over { $trackers } trackers.
```

Comment:
**question:** this string seems to be unused, can it be deleted?

Reply:
Oops, forgot to update this in the template. Good catch.